cpp/monitoring/counter.h

```diff
@@ -1,42 +1,92 @@
 #ifndef CERT_TRANS_MONITORING_COUNTER_H_
 #define CERT_TRANS_MONITORING_COUNTER_H_
 
+
+#include <gflags/gflags.h>
+#include <memory>
+#include <mutex>
 #include <string>
 
 #include "base/macros.h"
 #include "monitoring/metric.h"
+#include "monitoring/labelled_values.h"
 
 namespace cert_trans {
 
-
-// A metric which can only increase (e.g. total_requests_served).
-template <class... Labels>
+template <class... LabelTypes>
```

Comment:
No, but I did replace the contents of the file.
I *thought* I'd added that comment back, maybe I only added it to `gauge.h`.